packager/file/file.cc

```diff
@@ -287,7 +287,9 @@ bool File::WriteFileAtomically(const char* file_name,
 
   // Skip the warning message for memory files, which is meant for testing
   // anyway..
-  if (strncmp(file_name, kMemoryFilePrefix, strlen(kMemoryFilePrefix)) != 0) {
+  // Also check for http files, as they can't do atomic writes.
+  if (strncmp(file_name, kMemoryFilePrefix, strlen(kMemoryFilePrefix)) != 0
+      && strncmp(file_name, kHttpFilePrefix, strlen(kHttpFilePrefix)) != 0) {
```

Comment:
Needs to check `kHttpsFilePrefix` too.

Reply:
Oh, I didn't saw that var in `file.h`, so I missed it. 

Please take a look at the last commit. It's a quick commit, as I can't build shaka in my current device (I usually do test builds before commits), but should work anyways.